### PR TITLE
Added `Purchases.setLogLevel`

### PR DESCRIPTION
**Pull request by NachoSoto**
Fixes [CSDK-608].
Depends on https://github.com/RevenueCat/purchases-android/pull/753 and https://github.com/RevenueCat/purchases-hybrid-common/pull/301

[CSDK-608]: https://revenuecats.atlassian.net/browse/CSDK-608?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/plugin/plugin.ts [hunk 1]
```diff
@@ -898,13 +913,25 @@ class Purchases {
...
    */
   public static setDebugLogsEnabled(enabled: boolean): void {
     window.cordova.exec(null, null, PLUGIN_NAME, "setDebugLogsEnabled", [
       enabled,
     ]);
   }
 
+  /**
+   * Used to set the log level. Useful for debugging issues with the lovely team @RevenueCat.
+   * @param {boolean} enabled Enable or not debug logs
```
Comment (vegaro) [inline src/plugin/plugin.ts:926]: This should be updated

**Review by vegaro — APPROVED** [review]

File: src/plugin/plugin.ts (hunk 1 above)
Comment (aboedo) [reply src/plugin/plugin.ts:926]: also slight copy suggestion: 
```suggestion
   * @param {boolean} enabled Enable or disable debug logs
```

Comment (NachoSoto) [reply src/plugin/plugin.ts:933]: Oops thanks.

Comment (NachoSoto) [reply src/plugin/plugin.ts]: Fixed.

**Pull request merged**